nncf/torch/graph/graph_builder.py

```diff
@@ -134,4 +139,34 @@ def convert(dynamic_graph: DynamicGraph) -> PTNNCFGraph:
             )
 
         set_nodes_attributes_in_nncf_graph(nncf_graph)
+        # if traced_parameters:
+        #     propagate_is_shared_attribute_from_constant_nodes(nncf_graph)
         return nncf_graph
+
+
+def _propagate_true_for_is_shared_attribute(node: NNCFNode, graph: NNCFGraph):
+    """
+    Propagates the is_shared attribute through specific nodes in an NNCFGraph.
+
+    :param node: The start NNCFNode to process.
+    :param graph: The NNCFGraph instance.
+    """
+    node.attributes[NNCFNode.IS_SHARED_ATTR] = True
```

Comment:
What do you think about making `NNCFNode.IS_SHARED_ATTR` an a parameter of the function?

Reply:
done

Reply:
added `val` args

Reply:
I mean making the node attribute a parameter :) Like _Propagate_attribute(node, graph, True, NNCFNode.IS_SHAED_ATTR)

Reply:
Because for another attributes condition to propagation can be different.
This function can correctly propagate only is_shared attribute